core/repository/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id("java-library")
+    alias(libs.plugins.jetbrains.kotlin.jvm)
```

Comment:
`:core:repository` module is created as a pure library module to separate it from `:core:domain` (kotlin module) and `:core:data` (android module). This module includes only repository interface.